performance/buildSrc/src/main/kotlin/RunKotlinNativeTask.kt

```diff
@@ -45,12 +48,17 @@ open class RunKotlinNativeTask @Inject constructor(
         this.dependsOn += taskName
     }
 
+    private fun splitCommaSeparatedOption(arg: String, optionName: String) =
```

Comment:
As far as this function is exactly repeated may be better put it to Utils file. It can be even done as extension function for String.

Reply:
Moved to Utils file as extension function.